modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/cache/IdleVerify.java

```diff
@@ -346,6 +347,9 @@ private void cacheIdleVerifyV2(
                 IDLE_VERIFY_FILE_PREFIX + LocalDateTime.now().format(TIME_FORMATTER) + ".txt");
 
             try (PrintWriter pw = new PrintWriter(f)) {
+                if (client.state().state() == ClusterState.INACTIVE)
```

Comment:
Could you please clarify, why you uses `System.out.println` instead of `logger` which is available in `execute` method?

Reply:
It was decided to print the info re this into idle_verify-[date].txt which is specific for idle_verify, into control utility cmd output and not into control-utility.log itself.

Thus as per the current changes in this PR if cluster is inactive and has persistent data regions:
- exception "Exception: org.apache.ignite.IgniteException
Can not perform the operation because the cluster is inactive. Note, that the cluster is considered inactive by default if Ignite Persistent Store is used to let all the nodes join the cluster. To activate the cluster call Ignite.active(true)." is printed into idle_verify-[date].txt;
- "idle_verify does not work on an inactive cluster with persistence" is printed into control utility output not log itself.

Printing exceptions in IdleVerifyResultV2.print() is now limited to several ones with no IgniteException among them. It was decided to avoid creating smth like IgniteInactiveClusterWithPersistenceException.

Reply:
@sk0x50 we have a separate ticket for the logger issue https://issues.apache.org/jira/browse/IGNITE-17384

Reply:
@J-Bakuli , this section is for printing into file, it seems, that we should move it outside of `try` block.

Reply:
Thank you, I will remove that outside of try. Yes, it is for printing into idle_verify-[date].txt